src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.cs

```diff
@@ -1141,32 +1100,23 @@ internal string SpecificCultureName
                                                                             IcuGetThreeLetterWindowsLanguageName(_sRealName!);
 
         /// <summary>
-        /// Localized name for this language (Windows Only) ie: Inglis
-        /// This is only valid for Windows 8 and higher neutrals:
+        /// Localized name for this language
         /// </summary>
         private string LocalizedLanguageName
         {
             get
             {
-                if (_sLocalizedLanguage == null && !GlobalizationMode.Invariant)
+                string sLocalizedLanguage = NativeLanguageName;
```

Comment:
Nit: as long as we're touching the code, can we avoid the Hungarian notation? :)

Reply:
Sure. I'll fix that.